packages/react/src/patterns/sub-patterns/TableOfContents/TOCDesktop.js

```diff
@@ -64,25 +65,10 @@ const TOCDesktop = ({ menuItems, selectedId, updateState }) => {
     });
     const title = filteredItems[0].title;
     updateState(id, title);
-    document.querySelector(`a[name="${id}"]`).scrollIntoView({
-      behavior: 'smooth',
-      block: 'start',
-    });
-    triggerFocus(`a[name="${id}"]`);
+    const selector = `a[name="${id}"]`;
+    smoothScroll({ e: null, selector });
```

Comment:
See comment...

Reply:
@photodow If we did not pass an event with null argument in the object it is taking event as not null so and then looking for href. And I believe e: null is necessary that is how I am handling in utility.